p/db/ageGroup.js

```diff
@@ -9,22 +9,17 @@ const ageGroup = {
 		CHILD: 15,
 		YOUNG: 27,
 		ADULT: 65,
-		SENIOR: Infinity
-	}
+		SENIOR: Infinity,
+	},
 }
 
 const ageGroupFromAge = (age) => {
 	const {upperBoundOf} = ageGroup
-	if (age < upperBoundOf.BABY)
-		return ageGroup.BABY
-	if (age < upperBoundOf.CHILD)
-		return ageGroup.CHILD
-	if (age < upperBoundOf.YOUNG)
-		return ageGroup.YOUNG
-	if (age < upperBoundOf.ADULT)
-		return ageGroup.ADULT
-	if (age < upperBoundOf.SENIOR)
-		return ageGroup.SENIOR
+	if (age < upperBoundOf.BABY) return ageGroup.BABY
```

Comment:
I guess we can just always use `{`/`}` for conditionals & loop blocks.

Reply:
I handled this with commit 300a3e4.